openapi3.yaml

```diff
@@ -1,64 +1,101 @@
 openapi: 3.0.1
 info:
-  title: service-name
-  description: basic template for map colonies service
+  title: vector-tiles-rasterizer
+  description: A proxy server for rasterizing vector tiles
   version: 1.0.0
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
```

Comment:
it should match mapbox gl native license

Reply:
done.